src/snowflake/snowpark/_internal/analyzer/snowflake_plan.py

```diff
@@ -219,6 +223,14 @@ def wrap(*args, **kwargs):
                             missing_object_context = get_missing_object_context(
                                 top_plan, e.msg
                             )
+                        if (
+                            ("already exists" in e.msg)
```

Comment:
I'm just curious that would creating any kind of duplicate object (table, view, function, database etc.) trigger "already exists" error?

and your code would cover those cases as well?

e.g. 
```python
session.sql("create schema abc").collect()
session.sql("create schema abc").collect()
```

does your code also help with these cases?

Reply:
this is really helpful, thanks!